src/Chapter2.hs

```diff
@@ -780,12 +790,12 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
--- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
+listElementsLessThan :: Int -> [Int] -> [Int]
+listElementsLessThan x = filter (< x)
 
--- Can you eta-reduce this one???
+pairMul :: [Int] -> [Int] -> [Int]
 pairMul xs ys = zipWith (*) xs ys
```

Comment:
I see that you haven't eta-reduced this function? Do you need some help with this topic? :slightly_smiling_face:

Reply:
Oops missed it, will fix it 👍